### PR TITLE
chore: update eslit & husky

### LINKED ISSUE #35: Update `eslint` rules & remove `commitizen`

### PR DESCRIPTION
**Pull request by christoff-linde**
- update git pre-commit hooks 
The pre-commit hooks still created a commit even when the linter failed. Also, the pre-commit hook now uses `npx` (because `yarn` has been removed from the project)
- add `amplify` folder to `.gitignore`
- remove commitizen...for our own sanity

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -30,56 +16,73 @@ module.exports = {
...
+          requireLast: false,
+        },
+        singleline: {
+          delimiter: "comma",
+          requireLast: false,
+        },
+      },
+    ],
+    // note you must disable the base rule as it can report incorrect errors
+    camelcase: "off",
```
Comment (ksamson2) [inline .eslintrc.js]: Maybe we can turn `camelcase: "on",` ?

File: .eslintrc.js [hunk 2]
```diff
@@ -30,56 +16,73 @@ module.exports = {
...
+    "@typescript-eslint/no-explicit-any": "off",
```
Comment (ksamson2) [inline .eslintrc.js:34]: and this one off?

File: package.json [hunk 3]
```diff
@@ -100,18 +98,8 @@
...
-  "config": {
-    "commitizen": {
-      "path": "./node_modules/cz-conventional-changelog"
-    }
-  },
   "lint-staged": {
     "*.{js,ts,tsx,jsx}": "eslint --fix",
     "*.{js,ts,tsx,jsx,md,json}": "prettier --write"
-  },
-  "husky": {
```
Comment (ksamson2) [inline package.json:112]: How come we're removing husky?

**Review by ksamson2 — DISMISSED** [review]
I have minor questions, but none of them are blocking so feel free to just merge this and address the rest in a fast-follow

File: package.json (hunk 3 above)
Comment (christoff-linde) [reply package.json:112]: This was just for using husky with commitizen, husky is still part of the project :)

**Review by ksamson2 — APPROVED** [review]

**Pull request merged**